app/models/manageiq/providers/hawkular/middleware_manager/alert_manager.rb

```diff
@@ -2,8 +2,8 @@ module ManageIQ::Providers
   class Hawkular::MiddlewareManager::AlertManager
     require 'hawkular/hawkular_client'
 
-    def initialize(ems)
-      @alerts_client = ems.connect.alerts
+    def initialize(alerts_client)
+      @alerts_client = alerts_client
```

Comment:
Is this calling the `alerts_client` instance method of `ManageIQ::Providers::Hawkular::MiddlewareManager`?